packages/generate/lib/commands/plugin.js

```diff
@@ -216,23 +216,33 @@ After the first run the token gets stored in your system's keychain and will be
 				from: /\tregister_block_type\(.*\);\n/s,
 				to: '',
 			} );
+
+			await replace( {
+				files: pluginDir + '/webpack.config.js',
+				from: /\n\t\t'example-block-view': '\.\/blocks\/example\/view\.js',/s,
+				to: '',
+			} );
 		} );
 	}
 
 	// Rename files in local checkout.
 	await runStep( 'Renaming plugin files', 'Could not rename files.', async () => {
+		const files = [
+			pluginDir + '/README.md',
+			pluginDir + '/composer.json',
+			pluginDir + '/package.json',
+			pluginDir + '/phpcs.xml.dist',
+			pluginDir + '/webpack.config.js',
+			pluginDir + '/plugin.php',
+			pluginDir + '/inc/**/*.php',
+			pluginDir + '/assets/js/src/**/*.js',
```

Comment:
You did not get an error if the JS files were missing? Would this work too?

```suggestion
			pluginDir + '/assets/js/src/**/*.{js|json}',
```

There's also an `allowEmptyPaths` option: https://github.com/adamreisnz/replace-in-file#allow-emptyinvalid-paths

Reply:
Unfortunately not `Error: No files match the pattern: /Users/ulrich/Development/required/required-docker/swisscom-magazine/web/content/plugins/grappler-plugin/assets/js/src/**/*.{js|json}`

Reply:
Try `{js,json}` or `(js|json)`